<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
<summary errors="0" failures="2" finished="true" name="Test Run: 48" tests="7" total_time="Finished in: 4.00 secs."/>
<test location="Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py" status="ok" test="test_validate_input_is_decimal" time="0.00">
<stdout/>
<stderr/>
</test>
<test location="Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py" status="ok" test="test_validate_input_is_decimal2" time="0.00">
<stdout><![CDATA[Please enter a Decimal Number!
]]></stdout>
<stderr/>
</test>
<test location="Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py" status="ok" test="test_validate_input_is_decimal3" time="0.00">
<stdout><![CDATA[Please enter a Decimal Number!
]]></stdout>
<stderr/>
</test>
<test location="Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py" status="fail" test="test_validate_item_available" time="0.00">
<stdout/>
<stderr><![CDATA[File "Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py", line 39
in test_validate_item_available
    output = Validate.validate_item_available(restaurant_type)
E   TypeError: validate_item_available() missing 1 required positional argument: 'restaurant_name']]></stderr>
</test>
<test location="Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py" status="fail" test="test_validate_item_available2" time="0.00">
<stdout/>
<stderr><![CDATA[File "Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py", line 49
in test_validate_item_available2
    output = Validate.validate_item_available(restaurant_type)
E   TypeError: validate_item_available() missing 1 required positional argument: 'restaurant_name']]></stderr>
</test>
<test location="Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py" status="ok" test="test_validate_item_available3" time="0.18">
<stdout><![CDATA[//////////////////def get_food_items_availability/////////////////////

Starters
ChickenDrama
list_of_item_available
[]
No Rows Returned for that selected items
]]></stdout>
<stderr/>
</test>
<test location="Z:\FA3\Project-PyFood\PyFood\testcases\TestcasesModule3.py" status="ok" test="test_validate_view_category_items" time="0.04">
<stdout><![CDATA[////////////////def get_categories_fooditems///////////////////////

output validate_view_category
]]></stdout>
<stderr/>
</test>
<launch mode="run">
<launch_memento><![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="PyFood TestcasesModule3.py"/>
]]></launch_memento>
</launch>
</pydev-testsuite>
